pupil-spa/src/assets/shared-styles/elements/_colour-contrast.scss

```diff
@@ -0,0 +1,140 @@
+#page-modifications {
+  min-height: 100vh;
+  height: 100%;
+}
+
+@mixin colour-contrast($background-colour, $copy-colour) {
+  background-color: $background-colour;
+
+  #global-header {
+    background-color: $copy-colour !important;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    -ms-box-sizing: border-box;
+    box-sizing: border-box;
+
+    .header-wrapper, .header-logo {
+      background: none;
+    }
+
+    #proposition-name, #logo {
+      color: $background-colour !important;
+    }
+  }
+
+  #familiarisation-header-bar, .familiarisation-title {
+    color: $copy-colour !important;
+  }
+
+  #familiarisation-header-bar {
+    background-color: $background-colour !important;
+  }
+
+  h1, h2, h3, h4, p, strong, li, span, a, .timer, .remaining-time {
+    color: $copy-colour !important;
+  }
+
+  .back-link {
+    border-bottom-color: $copy-colour !important;
+
+    &:before {
+      border-right-color: $copy-colour !important;
+    }
+  }
+
+  .button, button, .numpad-button, .numpad-enter-button {
+    background-color: $copy-colour !important;
+    color: $background-colour !important;
+    box-shadow: none;
+  }
+
+  .answer {
+    box-shadow: 0 0 0 3px $copy-colour !important;
+    border: 1px solid $copy-colour !important;
+  }
+
+  .practice-box {
+    background-color: $copy-colour !important;
+    color: $background-colour !important;
+  }
+
+  .wait {
+
+    .background-fill {
+      fill: $background-colour !important;
+    }
+
+    .copy-stroke {
+      stroke: $copy-colour !important;
+    }
+
+    .copy-fill {
+      fill: $copy-colour !important;
+    }
+  }
+
+  //hide the blue header bar on all contrast pages
+  #global-header-bar {
+    display: none !important;
+  }
+}
+
+.colour-contrast-yob {
+  $copy-colour: #ff0;
+  $background-colour: #000;
+
+  @include colour-contrast($background-colour, $copy-colour);
+
+  // Some YOB colour are inverted
+  .numpad-button, .numpad-enter-button {
+    background-color: $background-colour !important;
+    color: $copy-colour !important;
+    border: 2px solid $copy-colour !important;
+  }
+
+  #global-header {
+    border: 2px solid $copy-colour !important;
+    background-color: $background-colour !important;
+
+    .header-wrapper, .header-logo {
+      background: none;
+    }
+
+    #proposition-name, #logo {
+      color: $copy-colour !important;
+    }
+  }
+
+  .practice-box {
+    background-color: $background-colour !important;
+    color: $copy-colour !important;
+    border: 2px solid $copy-colour !important;
+  }
+
+  label {
+    color: $copy-colour !important;
+  }
+
+  .button-start {
+    background: none;
+  }
+
+  // Fill eyes when YOB
```

Comment:
What does this mean?  Is there a screenshot?

Reply:
Heheh 😄 it's referring to the SVG, which is being styled according to what's needed.